src/bindings/python/src/pyopenvino/core/tensor.cpp

```diff
@@ -214,6 +214,31 @@ void regclass_Tensor(py::module m) {
             py::arg("begin"),
             py::arg("end"));
 
+    cls.def(py::init([](py::object& image) {
+                if (!py::isinstance(image, py::module::import("PIL.Image").attr("Image"))) {
+                    throw py::type_error("Input must be a PIL.Image.Image object");
+                }
+                auto np_array = py::module::import("numpy").attr("array")(image);
+                py::array array = np_array.cast<py::array>();
+
+                return Common::object_from_data<ov::Tensor>(array, false);
```

Comment:
I'd expect the default to reuse memory. But `shared_memory` is `false` here. One could explicitly copy if the needed, but there's no other way to make the memory shared on user side.

Reply:
I made some explorations and looks like it's impossible to introduce `shared_memory` mode for Pillow.Image. There is always be a copy, because pillow's data stored in chunks. Generally there is a good article from Pillow developer about the problem of sharing memory between numpy and pillow, but it is in Russian: https://habr.com/ru/articles/545850/